src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -75,14 +79,164 @@ index c8a6fef58d31..7d01c195791e 100644
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-1           │ Fixes issue 1                                   │ Optional │ Applied     │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-2           │ Fixes issue 2                                   │ Optional │ Not applied │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-3           │ Fixes issue 3                                   │ Optional │ Not applied │ Required patches:               ║
+║                │                                                 │          │             │  - MC-2                         ║
+║                │                                                 │          │             │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ MC-3-V2        │ Updated fix for issue 3, replaces MC-3 patch    │ Optional │ N/A         │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-cms           ║
+╚════════════════╧═════════════════════════════════════════════════╧══════════╧═════════════╧═════════════════════════════════╝
+Magento 2 Enterprise Edition, version 2.3.5.0
+```
+
+The status table contains the following types of information:
+
+-  **Type**:
+
+   -  `Optional`—All individual patches from the MQP package and the [Magento Cloud Patches]({{ site.baseurl }}/cloud/project/project-patch.html) package are optional for {{ site.data.var.ee }} and {{ site.data.var.ce }} installations.
+   -  `Deprecate`d—The individual patch is marked as deprecated by Magento and we recommend reverting it if you have applied it. After you revert a deprecated patch, it will no longer be displayed in the status table.
+
+-  **Status**:
+
+   -  `Applied`—The patch has been applied.
+   -  `Not applied`—The patch has not been applied.
+   -  `N/A`—The status of the patch cannot be defined due to conflicts.
+
+-  **Details**:
+
+   -  `Affected components`—The list of affected Magento modules.
+   -  `Required patches`—The list of required patches (dependencies).
```

Comment:
What makes a patch required?  

All patches delivered by Cloud Patches package are required.
All patches delivered by MQP are "individual patches" as defined 
Does MQP deliver required patches using Cloud Patches set? 

Maybe `required` isn't a category for Magento open source and enterprise?

Reply:
Required patches are patches that must be applied for the current patch to work properly. It is a dependency. Defined by the published in the MQP configuration file.
https://wiki.corp.magento.com/display/SUP/Advanced+Patch+Configuration+Reference

Reply:
Clarified in 30a076c7db8f13dcfe631e90168b586ab392f60c